lib/engine-addon.js

```diff
@@ -1043,7 +1044,8 @@ module.exports = {
           escape(JSON.stringify(engineConfig)) +
           '" />';
 
-        return content;
+        childrenConfig.push(content);
```

Comment:
Can we guard this to avoid pushing the `<meta>` when `engineConfig` is effectively empty (e.g. no reason to push the element)?

Reply:
Done